frontend/src/state.ts

```diff
@@ -1,7 +1,7 @@
-import { Types, Maybe, SortedCollection } from '@dotstats/common';
+import { Types, Maybe, SortedCollection } from './common';
 import { Column } from './components/List';
 
-export const PINNED_CHAIN = 'Kusama';
+export const PINNED_CHAIN = 'Kusama CC3';
```

Comment:
AFAIK, the network was renamed recently from `Kusama CC3` to `Kusama` so this change does not look ok.

Reply:
Yup :point_up: